### PR TITLE
ctime: Drop 32-/64-bit distinction on Linux

### PR DESCRIPTION
**Pull request by wking**
We added the explicit `int64` casts for 32-bit builds in 35e1ad78 (#324), but the explicit casts work fine on 64-bit systems too.

### CODE REVIEW

File: pkg/ctime/ctime_linux_64.go [hunk 1]
```diff
@@ -1,14 +0,0 @@
-// +build !arm,!386,linux
```
Comment (TomSweeneyRedHat) [inline pkg/ctime/ctime_linux_64.go]: Should this still be "// +build linux"?

Comment (mheon) [reply pkg/ctime/ctime_linux_64.go]: If the file is gone, I don't think we need a tag for it

**Comment by mheon** [comment]
LGTM

File: pkg/ctime/ctime_linux_64.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply pkg/ctime/ctime_linux_64.go]: @mheon, DOH! you're right.  Too many reviews today.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by wking** [comment]
All green :).

**Comment by wking** [comment]
> jwhonce assigned wking...

Not much for me to do here, just need an approver to pull the trigger.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=mheon

**Comment by mheon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**